buildpacks/maven/src/main.rs

```diff
@@ -238,10 +267,15 @@ impl Buildpack for MavenBuildpack {
                         .iter(),
                     ),
                 )
-                .envs(&mvn_env),
-            MavenBuildpackError::MavenBuildIoError,
-            MavenBuildpackError::MavenBuildUnexpectedExitCode,
-        )?;
+                .envs(&mvn_env);
+
+            output::run_command(
+                command,
+                true,
+                MavenBuildpackError::MavenBuildIoError,
+                |output| MavenBuildpackError::MavenBuildUnexpectedExitCode(output.status),
+            )
+        })?;
```

Comment:
Originally requested on the incorrect line https://github.com/heroku/buildpacks-jvm/pull/745#pullrequestreview-2476776597

Specific requests:

- Please either stream the output as it happens or add it to the error information (but not both). 
- Please add the command being run to the error as well (it doesn't look like it's emitted)
- If this command could take a variable amount of time (such as needing the network) please add background printing to let the user know we're still waiting for something to finish.

Reply:
I moved this feedback to a "lster" issue https://github.com/heroku/buildpacks-jvm/issues/762